### PR TITLE
changing implementation of SCCs deployment

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**
Unlike in upstream, in downstream both worker and topology daemonsets
require dedicated SSCs. Unfortunatly, OLM does not support deploying
SCCs as part of a bundle. Therefore, in downstream, the NFD operator
will deploy SCCs as part of reconciliation loop

### CODE REVIEW

File: internal/daemonset/mock_daemonset.go [hunk 1]
```diff
@@ -12,9 +12,9 @@ import (
 	context "context"
 	reflect "reflect"
 
-	v10 "github.com/openshift/cluster-nfd-operator/api/v1"
+	v1 "github.com/openshift/cluster-nfd-operator/api/v1"
 	gomock "go.uber.org/mock/gomock"
-	v1 "k8s.io/api/apps/v1"
+	v10 "k8s.io/api/apps/v1"
```
Comment (chr15p) [inline internal/daemonset/mock_daemonset.go:17]: the same for the v1 / v10 swap here, I dont see that it makes much difference to the functionality, so fine, but its not clear to me how it relates to scc's

File: internal/configmap/mock_configmap.go [hunk 2]
```diff
@@ -12,9 +12,9 @@ import (
 	context "context"
 	reflect "reflect"
 
-	v10 "github.com/openshift/cluster-nfd-operator/api/v1"
+	v1 "github.com/openshift/cluster-nfd-operator/api/v1"
 	gomock "go.uber.org/mock/gomock"
-	v1 "k8s.io/api/core/v1"
+	v10 "k8s.io/api/core/v1"
```
Comment (chr15p) [inline internal/configmap/mock_configmap.go:17]: are we just swapping the v1 and v10 labels around here?  it doesn't look like it should make any difference so I'm not sure I understand why we're doing it.

File: api/v1/nodefeaturerules_types.go
Comment (chr15p) [inline api/v1/nodefeaturerules_types.go:1]: Does this have anything to do with SCCs?   I assume its just left over from the refactor and is all defined in the operand, but (imho) removing it should probably be a separate PR.

File: Makefile [hunk 3]
```diff
@@ -153,7 +153,7 @@ clean-labels:
 # Generate code
 generate: controller-gen mockgen
 	$(CONTROLLER_GEN) object:headerFile="utils/boilerplate.go.txt" paths="./..."
-	$(GO_CMD) generate ./...
+	go generate ./...
```
Comment (chr15p) [inline Makefile:156]: we should probably either use $(go_CMD) everywhere, or hardcode "go" everywhere rather than the mix we have at the moment, but that's a problem for another day :)

**Review by chr15p — APPROVED** [review]
looks good to me, I just think theres a couple of changes that should have ben separate PRs, if only to make it easier on me reviewing them!

File: internal/daemonset/mock_daemonset.go (hunk 1 above)
Comment (yevgeny-shnaidman) [reply internal/daemonset/mock_daemonset.go:17]: This is the result of running generate command...

File: internal/configmap/mock_configmap.go (hunk 2 above)
Comment (yevgeny-shnaidman) [reply internal/configmap/mock_configmap.go:17]: This is the result of running generate command...

File: Makefile (hunk 3 above)
Comment (yevgeny-shnaidman) [reply Makefile:156]: I will fix it in a different PR
